week-1/L-functions-nested/exercise2.js

```diff
@@ -1,5 +1,19 @@
+function Greeting(hello) {
```

Comment:
Same here about the Greeting, better lowercase. Also, you have an argument in your function declaration, that you never use and you reassigning the argument variable with a static value. Thinks about refactoring this and we can discuss it in the class.